packages/paste-website/src/pages/components/alert/index.mdx

```diff
@@ -321,20 +297,6 @@ In general, the persistence and behavior on scroll of an alert should match the
   />
 </DoDont>
 
-## Anatomy
```

Comment:
question: is deletion of the anatomy tables intentional with these component updates?

Reply:
It started going along with the component updates because the values are changing, and instead of updating them we've been removing them altogether because it's hard to maintain them: https://github.com/twilio-labs/paste/pull/2938#discussion_r1070039495